src/pages/HomePage/index.tsx

```diff
@@ -1,13 +1,34 @@
+import * as RemoteData from '@devexperts/remote-data-ts';
 import * as React from 'react';
-import { Link } from 'react-router-dom';
+import { Link, useHistory } from 'react-router-dom';
 
-import { useUserContext } from '~context/userContext';
 import * as Routes from '~lib/routes';
 
+import { useAppSelector } from '../../redux/Store';
+import { Rolle } from '../../types/LoggedInUser';
+
 import styles from './homePage.module.less';
 
 const HomePage = () => {
-    const context = useUserContext();
+    const loggedInUser = useAppSelector((s) => s.me.me);
+    const history = useHistory();
+
+    React.useEffect(() => {
+        if (RemoteData.isSuccess(loggedInUser) && loggedInUser.value.roller.length === 1) {
+            switch (loggedInUser.value.roller[0]) {
```

Comment:
Man kan väl ha flera roller? 😊

Reply:
Jepp, det er derfor jeg bare har lagt inn denne redirecten dersom man har kun én rolle.

Reply:
Har forresten oppdatert dette til å funke fint for de som er både saksbehandler og attestant.